Packs/Jira/Integrations/JiraV3/JiraV3.yml

```diff
@@ -419,7 +419,7 @@ script:
       name: issue_id
     - description: The issue key (Issue ID or key is required).
       name: issue_key
-    - description: 'The issue object (in JSON format). Using this argument will override the other arguments. For example {"fields":{"customfield_10037":"field_value"}}.'
+    - description: 'The issue object (in JSON format). This argument is for advanced users, as when utilizing this argument, one cannot use other arguments (it will raise an error) except the `issue_id` `issue_key` `action` `transition` and `status` and must input all required arguments into the issue_json. In addition, when issue_json is used, it is not possible to use the project name but only the project key. For example {"fields":{"customfield_10037":"field_value", "summary": "some summary", "project": {"key": "<project_key>"}}}. For more information about this argument, see https://developer.atlassian.com/server/jira/platform/jira-rest-api-example-create-issue-7897248/.'
```

Comment:
```suggestion
    - description: 'The issue object (in JSON format). This argument is for advanced users. When used, one cannot use other arguments (it will raise an error) except the `issue_id`, `issue_key`, `action`, `transition`, and `status`. You must input all required arguments into the issue_json. In addition, when issue_json is used, it is not possible to use the project name but only the project key or ID. For example {"fields":{"customfield_10037":"field_value", "summary": "some summary", "project": {"key": "<project_key>"}}}. For more information about this argument, see https://developer.atlassian.com/server/jira/platform/jira-rest-api-example-create-issue-7897248/.'
```